docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -42,41 +58,60 @@ To add another catalog:
 For example, if you name the property file ``sales.properties``, Trino uses the
 configured connector to create a catalog named ``sales``.
 
-Querying SQL Server
--------------------
+.. _sqlserver-type-mapping:
+
+Type mapping
+------------
 
-The SQL Server connector provides access to all schemas visible to the specified user in the configured database.
-For the following examples, assume the SQL Server catalog is ``sqlserver``.
+Trino supports the following SQL Server data types:
 
-You can see the available schemas by running ``SHOW SCHEMAS``::
+==================================  ===============================
+SQL Server Type                     Trino Type
+==================================  ===============================
+``bigint``                          ``bigint``
+``smallint``                        ``smallint``
+``int``                             ``integer``
+``float``                           ``double``
+``char(n)``                         ``char(n)``
+``varchar(n)``                      ``varchar(n)``
+``date``                            ``date``
+``datetime2(n)``                    ``timestamp(n)``
+==================================  ===============================
 
-    SHOW SCHEMAS FROM sqlserver;
+Complete list of `SQL Server data types
+<https://msdn.microsoft.com/en-us/library/ms187752.aspx>`_.
 
-If you have a schema named ``web``, you can view the tables
-in this schema by running ``SHOW TABLES``::
+.. include:: jdbc-type-mapping.fragment
 
-    SHOW TABLES FROM sqlserver.web;
+SQL support
```

Comment:
We will need to add more details in this section in a future PR. For now I just used the old content from limitations section